README.md

```diff
@@ -1,2 +1,6 @@
-# Deploying Machine Learning Models
-For the documentation, visit the course on Udemy.
+# Deployment of Machine Learning Models
+Accompanying repo for the online course Deployment of Machine Learning Models.
+
+For the documentation, visit the [course on Udemy](https://www.udemy.com/deployment-of-machine-learning-models/?couponCode=TIDREPO).
+
+![Deployment of Machine Learning Models](https://github.com/trainindata/deploying-machine-learning-models/blob/master/Logo.png)
```

Comment:
@solegalli this logo won't work ("blob" shouldn't be in url, also easier to use a relative path), and looks a bit big without styling. I prepared a copy here if you want to compare: https://github.com/ChristopherGS/ud-draft)

Suggest we remove.